RevenueCat/Scripts/Purchases.cs

```diff
@@ -499,40 +609,106 @@ public void SyncPurchases()
     }
 
+    /// <summary>
+    /// iOS only. Callback for the <see cref="Purchases.CheckTrialOrIntroductoryPriceEligibility"/> method.  
+    /// </summary>
+    /// <param name="products"> A Dictionary mapping product identifiers to their eligibility status,
+    /// as <see cref="IntroEligibility"/> objects.</param>
+    public delegate void CheckTrialOrIntroductoryPriceEligibilityFunc(Dictionary<string, IntroEligibility> products);
+
     private CheckTrialOrIntroductoryPriceEligibilityFunc CheckTrialOrIntroductoryPriceEligibilityCallback { get; set; }
 
+    /// 
+    /// <summary>
+    /// iOS only. Computes whether or not a user is eligible for the introductory pricing period of a given product.
+    /// You should use this method to determine whether or not you show the user the normal product price or
+    /// the introductory price. This also applies to trials (trials are considered a type of introductory pricing).
+    /// <seealso href="https://docs.revenuecat.com/docs/ios-subscription-offers"/>.
+    /// </summary>
+    ///
+    /// <remarks>
+    /// Note: If you're looking to use Promotional Offers instead,
+    /// use <see cref="GetPromotionalOffer"/>.
+    /// </remarks>
+    ///
+    /// <remarks>
+    /// Note: Subscription groups are automatically collected for determining eligibility. If RevenueCat can't
+    /// definitively compute the eligibility, most likely because of missing group information, it will return
+    /// <see cref="IntroEligibilityStatus.IntroEligibilityStatusUnknown"/>.
+    /// The best course of action on unknown status is to display the non-intro
+    /// pricing, to not create a misleading situation. To avoid this, make sure you are testing with the latest
+    /// version of iOS so that the subscription group can be collected by the SDK.
+    /// </remarks>
+    ///
+    /// <param name="products"> The <see cref="StoreProduct"/>  for which you want to compute eligibility.</param>
+    /// <param name="callback"> The <see cref="CheckTrialOrIntroductoryPriceEligibilityCallback"/> callback. </param>
     public void CheckTrialOrIntroductoryPriceEligibility(string[] products,
         CheckTrialOrIntroductoryPriceEligibilityFunc callback)
     {
         CheckTrialOrIntroductoryPriceEligibilityCallback = callback;
         _wrapper.CheckTrialOrIntroductoryPriceEligibility(products);
     }
 
+    ///
+    /// <summary>
+    /// Invalidates the cache for customer information.
+    /// </summary>
+    /// 
+    /// <remarks>Most apps will not need to use this method; invalidating the cache can leave your app in an invalid state.
+    /// Refer to https://docs.revenuecat.com/docs/purchaserinfo#section-get-user-information
+    /// for more information on using the cache properly.
+    /// </remarks>
+    ///
+    /// This is useful for cases where customer information might have been updated outside of the app, like if a
+    /// promotional subscription is granted through the RevenueCat dashboard.
```

Comment:
Will this text outside the `<summary>` tag be included in the docs?

Reply:
yeah, it'll still be there. I have yet to see how it looks when you generate to a website, but for now in the IDE it does work. My hope is that it'll render into a separate section, like in iOS, where you have the summary and the "discussion" as separate things ([example here](https://revenuecat.github.io/purchases-ios-docs/4.9.0/documentation/revenuecat/purchases/configure(with:)-6oipy))